pkgs/games/runelite/default.nix

```
@@ -1,17 +1,17 @@
 { stdenv, fetchurl, makeWrapper, jre, makeDesktopItem, lib }:
 
-  stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "runelite";
-  version = "1.6.0";
+  version = "2.0.0";
 
   src = fetchurl {
     url = "https://github.com/runelite/launcher/releases/download/${version}/RuneLite.jar";
-    sha256 = "0q2xx0wrnlg5mrv8nnmnh300r8mqfm8k2p028m7mr09kn18xvkzx";
+    sha256 = "0q0x8g43ggkvp5fgnymgygx59xfhnyyrhpi6ha333gqg97rr0rvw";
   };
 
   icon = fetchurl {
     url = "https://github.com/runelite/runelite/raw/master/runelite-client/src/main/resources/runelite.png";
```

would you mind updating the url to use the tag url?
```suggestion
     url = "https://github.com/runelite/runelite/raw/${version}/runelite-client/src/main/resources/runelite.png";
```